vignettes/articles/sliding.Rmd

```diff
@@ -309,6 +309,11 @@ k_week_version_aware <- function(ahead = 7, version_aware = TRUE) {
         args_list = arx_args_list(ahead = ahead)
       ) %>%
         extract2("predictions"),
+      # Both `before` in `epi_slide` and `n_training` in `arx_forecaster()`
+      # Controls number of traning samples forecaster can see
+      # `before` first controls the number of samples at a particular step 
```

Comment:
`n_training` isn't an argument to `arx_forecaster()` but to `arx_args_list()`. 

I think it's OK to document `before` here like `# controls how many days in the `epi_df` are used (see ...)`.

But the rest is really details that should go in the text of the Vignette with additional details.

Reply:
Yes, thanks for pointing out `n_training`, I wasn't most precise there. 

Let's drop the additional text on `before` and `n_training` to maybe an example with more details.

Reply:
I am actually not too sure about the 

> n_training
> Integer. An upper limit for the number of rows per key that are used for training (in the time unit of the epi_df).

What is the key referring to here? Is it the key in `quantile_by_key`?

Reply:
`epi_df`'s can contain `additional_keys` besides just `geo_value` (say age bins).